teku/src/main/java/tech/pegasys/teku/cli/options/MetricsOptions.java

```diff
@@ -121,28 +121,48 @@ public class MetricsOptions {
       MetricsConfig.DEFAULT_BLOCK_PRODUCTION_AND_PUBLISHING_PERFORMANCE_ENABLED;
 
   @Option(
-      names = {"--Xmetrics-block-production-timing-tracking-warning-threshold"},
+      names = {"--Xmetrics-block-production-timing-tracking-warning-local-threshold"},
       hidden = true,
       showDefaultValue = Visibility.ALWAYS,
       paramLabel = "<INTEGER>",
       description =
-          "The time (in ms) at which block production is to be considered 'slow'. If set to 100, block production taking at least 100ms would raise a warning.",
-      fallbackValue = "true",
-      arity = "0..1")
-  private int blockProductionPerformanceWarningThreshold =
-      MetricsConfig.DEFAULT_BLOCK_PRODUCTION_PERFORMANCE_WARNING_THRESHOLD;
+          "The time (in ms) at which block production using a local flow is to be considered 'slow'. If set to 100, block production taking at least 100ms would raise a warning.",
```

Comment:
I think we can fix the description here: it is not the time taken but it is tame taken from the beginning of the slot (which is not just processing time).

(applies to all 4)